org.mozilla.vpn.yml

```diff
@@ -0,0 +1,53 @@
+app-id: org.mozilla.vpn
+runtime: org.kde.Platform
+runtime-version: 6.8
```

Comment:
use 6.9?

Reply:
We haven't tested the application so far on Qt 6.9, and `qt6.8` is the current default branch in https://invent.kde.org/packaging/flatpak-kde-runtime so I decided to go with that one. That being said, I can spend some time testing things out on Qt6.9 and update to that if it all seems good.

Reply:
The default branch is just omission, most apps moved to 6.9 alrady